src/Uno.Toolkit.UI/Controls/NavigationBar/NavigationBar.xaml

```diff
@@ -1443,8 +1444,7 @@
 		<Setter Property="OverflowButtonVisibility" Value="Collapsed" />
 		<Setter Property="IsDynamicOverflowEnabled" Value="False" />
 		<Setter Property="MainCommandStyle" Value="{StaticResource DefaultMainCommandStyle}" />
-		<mobile:Setter Property="utu:SafeArea.Insets"
-					   Value="Top" />
+		<Setter Property="utu:SafeArea.Insets" Value="Top" />
```

Comment:
Will not this break other targets? Like Desktop?

Reply:
No, it should be ok, on other targets the evaluated insets will always stay 0